src/components/Reports/ReportEdit.tsx

```diff
@@ -39,8 +43,20 @@ export default function ReportEdit(): JSX.Element {
     } else {
       snackbar.toastError(strings.GENERIC_ERROR, strings.REPORT_COULD_NOT_OPEN);
     }
+
+    setCheckUserInterval(setInterval(getReport, 60000));
   }, [reportIdInt, snackbar]);
 
+  useEffect(() => {
+    if (report && user && report?.lockedByUserId !== user?.id) {
+      if (checkUserInterval) {
+        clearInterval(checkUserInterval);
+        setCheckUserInterval(undefined);
+      }
+      history.push(APP_PATHS.REPORTS_VIEW.replace(':reportId', reportId).concat('?invalidEditor=true'));
```

Comment:
we should probably let user know before redirecting? a popup.. user hits ok.. then redirect?
i understand we are showing a popup info on the next page instead...

Reply:
I was thinking of just the redirect for an initial pass when I described to @constanzauanini. If we want to have a bit more interaction, like maybe popup confirmation as you suggest, or allow to steal the lock back, we can. I agree that this could be a very jarring UX.

Reply:
Got it. A simpler initial pass does sound good. Stealing locks is interesting.. users battling it out 🍿 
At some later point yes we should redirect after informing the user.

Reply:
btw, if you think we won't revisit this to cleanup later... my suggestion is to clean this up now.. while we are working on this anyway.

Reply:
So we don't want to show the toast message and show a message here?

Reply:
Or just create a ticket to revisit later.